ci/google.yml

```
@@ -7,69 +7,71 @@
 # 2. Set up secrets in your workspace: GKE_PROJECT with the name of the project, GKE_EMAIL with the service account email, GKE_KEY with the Base64 encoded JSON service account key (https://github.com/GoogleCloudPlatform/github-actions/tree/docs/service-account-key/setup-gcloud#inputs).
 #
 # 3. Change the values for the GKE_ZONE, GKE_CLUSTER, IMAGE, REGISTRY_HOSTNAME and DEPLOYMENT_NAME environment variables (below).
+#
+# For more support on how to run the workflow, please visit https://github.com/GoogleCloudPlatform/github-actions/tree/master/example-workflows/gke
 
 name: Build and Deploy to GKE
 
 on:
   release:
     types: [created]
 
-# Environment variables available to all jobs and steps in this workflow
 env:
-  GKE_PROJECT: ${{ secrets.GKE_PROJECT }}
-  GKE_EMAIL: ${{ secrets.GKE_EMAIL }}
-  GITHUB_SHA: ${{ github.sha }}
-  GKE_ZONE: us-west1-a
-  GKE_CLUSTER: example-gke-cluster
-  IMAGE: gke-test
-  REGISTRY_HOSTNAME: gcr.io
-  DEPLOYMENT_NAME: gke-test
+  PROJECT_ID: ${{ secrets.GKE_PROJECT }}
+  GKE_CLUSTER: cluster-1    # TODO: update to cluster name
+  GKE_ZONE: us-central1-c   # TODO: update to cluster zone
+  DEPLOYMENT_NAME: gke-test # TODO: update to deployment name
+  IMAGE: static-site
 
 jobs:
   setup-build-publish-deploy:
     name: Setup, Build, Publish, and Deploy
     runs-on: ubuntu-latest
-    steps:
 
+    steps:
     - name: Checkout
       uses: actions/checkout@v2
 
     # Setup gcloud CLI
     - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```

@imjohnbo Please correct me if I am wrong, but the recommendation was to pin this to a release right?

---

That's right, it looks like the latest release is `0.1.3`, @bharathkkb 👍

---

It would be best to reference a major version like `v1` [[docs](https://docs.github.com/en/actions/creating-actions/about-actions#using-release-management-for-actions)]

---

I don't think we have a v1 tagged release. Would it be recommended to pin it to 0.1.3 or keep it master?
```suggestion
    - uses: GoogleCloudPlatform/github-actions/setup-gcloud@v0.1.3
```

---

Yeah I think `0.1.3` is fine for now as a replacement for `master` but it will quickly become out of date as new features (and actions) are added, which is why we recommend pinning to a major version tag like `v1` (`v` is an optional recommended prefix) and moving that tag to the Gif ref of the current release.

```suggestion
    - uses: GoogleCloudPlatform/github-actions/setup-gcloud@0.1.3
```